sharedbuffers/_version.py

```diff
@@ -1 +1 @@
-__version__ = '0.4.2'
\ No newline at end of file
+__version__ = '0.4.3'
```

Comment:
shouldn't this file have a new line?

Reply:
It's autogenerated by setup.py, if it's missing we can make the change in setup.py to add the newline.

Reply:
ok